Mathlib/FieldTheory/RatFunc/Degree.lean

```diff
@@ -53,13 +53,15 @@ theorem intDegree_one : intDegree (1 : RatFunc K) = 0 := by
 @[simp]
 theorem intDegree_C (k : K) : intDegree (C k) = 0 := by
   rw [intDegree, num_C, natDegree_C, denom_C, natDegree_one, sub_self]
-set_option linter.uppercaseLean3 false in #align ratfunc.int_degree_C RatFunc.intDegree_C
+set_option linter.uppercaseLean3 false in
+#align ratfunc.int_degree_C RatFunc.intDegree_C
 
 @[simp]
 theorem intDegree_X : intDegree (X : RatFunc K) = 1 := by
   rw [intDegree, num_X, Polynomial.natDegree_X, denom_X, Polynomial.natDegree_one,
     Int.ofNat_one, Int.ofNat_zero, sub_zero]
-set_option linter.uppercaseLean3 false in #align ratfunc.int_degree_X RatFunc.intDegree_X
+set_option linter.uppercaseLean3 false in
```

Comment:
Are you sure this is the recommended style?

Reply:
It is certainly the style that is used *every other time*!  :smile: